src/components/MembershipGroupListItemEditable.tsx

```diff
@@ -30,7 +29,7 @@ export const MembershipGroupListItemEditable = ({ membership }: MembershipGroupL
                 {membership.percentage}
             </Text>
 
-            {user?.role === UserRole.ADMIN && <MembershipEditMenu membership={membership} />}
+            <MembershipEditMenu membership={membership} isEditable={session.data?.user.role === UserRole.ADMIN} />
```

Comment:
that probably should be an access check. Also can we just make a component `RoleInterface` which hides children when access is limited?

Reply:
Added new issue to get rid of that role check https://github.com/taskany-inc/crew/issues/596